packages/acceptance-tests/pkg-tests-specs/sources/commands/pack.test.js

```diff
@@ -593,5 +593,22 @@ describe(`Commands`, () => {
         expect(xfs.existsSync(`${tmpDir}/test.tgz`)).toEqual(true);
       }),
     );
+
+    test(
```

Comment:
Are there other areas you would recommend me adding a test for?  One thing I noticed was that `yarn install` was removing the files array as a result of the bug, but since it's related to this and this test would cover that I'm not sure if it makes sense to also test that.

Reply:
Just this test is perfect 👍